localbuild.xml

```diff
@@ -434,7 +434,7 @@
   <project path="external/python/parse_type" name="platform/external/python/parse_type" groups="vts,pdk" />
```

Comment:
nitpick: commit message should read `cloning` instead of `cloneing` (the typo cloned itself:)